README.md

```diff
@@ -1770,6 +1772,7 @@ Update Time, five active automations, webhooks.
   * [Hashnode](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
   * [Medium](https://medium.com/) — Get more thoughtful about what matters to you.
   * [AyeDot](https://ayedot.com/) — Share your ideas, knowledge, and stories with the world for Free in the form of Modern multimedia short-format Miniblogs.
+  * [Stomod](https://stomod.com/) — Notion docs to branded, SEO-optimized blog in minutes!
```

Comment:
please mention a bit whats free - especially mention that it doesnt do custom domains for free.  Also undo the change to the contents please